src/ui/Combobox/ComboboxValues.ts

```diff
@@ -151,6 +151,37 @@ export class ComboboxValues {
     this.combobox.clearAll();
   }
 
+  private onScroll() {
+    const scrollEndReached = this.element.scrollTop + this.element.clientHeight >= this.element.scrollHeight;
```

Comment:
Should we load more suggestions when the very end is reached or a little bit before (e.g., 2 suggestions before the end)?

I personally don't feel strongly about either method, but I'm curious what would be everyone's opinion.

Reply:
same, this was more simple and works quite well

Reply:
The thinking these days is elements should be loaded at the latest possible point, while trying to ensure that a user does not wait, or experience a delay.

I think for JSUI the current approach is good and enough.
For Atomic and e-commerce use-cases, preloading results will be important if we want to deliver the best experience.